.github/workflows/daily.yml

```diff
@@ -148,20 +182,6 @@ jobs:
       - name: Checkout branch or tag
         uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
 
-      # this step needs to be run here while we have `mathlib4` checked out,
-      # or else `gh run view` will try to look for runs in `mathlib4-nightly-testing`
-      - name: Get URLs
-        id: urls
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          # adapted from https://cmbuckley.co.uk/blog/2024/04/09/deep-links-to-github-actions-job-logs/
-          mathlib_test_executable_url=$(gh run view ${{ github.run_id }} --json jobs --jq '
-            .jobs[] | select(.name == "check-lean4checker (${{ matrix.branch_type }})")
```

Comment:
This `check-lean4checker` was incorrect, causing the URLs in the zulip message to be blank. It should have been `check-mathlib_test_executable`. This is fixed in the refactored version below.